modules/renter/contractor/contractor_test.go

```diff
@@ -394,6 +395,8 @@ func TestAllowanceSpending(t *testing.T) {
 	}
 	newReportedSpending := c.PeriodSpending()
 	if reflect.DeepEqual(newReportedSpending, reportedSpending) {
+		log.Print(reportedSpending)
+		log.Print(newReportedSpending)
```

Comment:
I think this is failing because of the comparison of `types.Currency`, which contains a slice. If you print with `log.Printf("%+v", reportedSpending)` you may be able to see the precise difference. But at any rate, it seems like we need to write an `Allowance.Equals(Allowance)` method to manually compare allowances.